### PR TITLE
Fix/snackbar

### PR DESCRIPTION
**Pull request by gmbarroso**
## Context
This PR corrects some issues regarding the Snackbar component. From bug fixes to storybook updates

## Checklist
- [x] Remove the props `actions`.
- [x] Add Snackbar to the styleguide
- [x] Add the Snackbar component with event action to the index.md file
- [x] Update the example files to it correct location (stories)

## Deploy Notes
- `git clone` in the repository
- `yarn install` to install dependencies
- `yarn storybok` to run the application and `yarn styleguide` to see the `former-kit`doc

### CODE REVIEW

File: stories/Snackbar/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { storiesOf } from '@storybook/react'
 import IconClose from 'emblematic-icons/svg/ClearClose24.svg'
 
 import Snackbar from '../../src/Snackbar'
-import SnackbarWithState from '../../src/Snackbar/examples/SnackbarWithState'
+import SnackbarWithState from '../Snackbar/SnackbarWithState'
```
Comment (rsmelo) [inline stories/Snackbar/index.js]: `SnackbarWithState` is in the same directory, it is not necessary to `..` up a level
```suggestion
import SnackbarWithState from './SnackbarWithState'
```

File: src/Snackbar/index.md [hunk 2]
```diff
@@ -35,3 +35,17 @@ Success
...
 ```
+
+Snackbar With Event
+```jsx
+const IconClose = require('emblematic-icons/svg/ClearClose24.svg').default;
+
+<Snackbar
+  action=""
+  icon={<IconClose height={12} width={12} />}
+  onDismiss={action('dismiss')}
```
Comment (rsmelo) [inline src/Snackbar/index.md]: there is an error with action
Screen Shot 2019-03-12 at 20 29 59

Comment (uselessdev) [reply src/Snackbar/index.md]: Eu sugiro, adicionar uma função que não faz nada, `onDismiss={null}` actions fazem parte do storybook e não exite nenhum motivo especial para elas existirem aqui

File: stories/Snackbar/index.js (hunk 1 above)
Comment (gmbarroso) [reply stories/Snackbar/index.js]: You are right, @rsmelo . Consider it done. :smile:

File: src/Snackbar/index.md (hunk 2 above)
Comment (gmbarroso) [reply src/Snackbar/index.md]: Consider it done.

**Pull request merged**